compiler/fir/resolve/src/org/jetbrains/kotlin/fir/resolve/transformers/body/resolve/FirExpressionsResolveTransformer.kt

```diff
@@ -374,64 +377,90 @@ open class FirExpressionsResolveTransformer(transformer: FirBodyResolveTransform
             assignOperatorCall.transformSingle(this, ResolutionMode.ContextDependent)
         }
         val assignCallReference = resolvedAssignCall.calleeReference as? FirNamedReferenceWithCandidate
-        val assignIsError = assignCallReference?.isError ?: true
+        val assignIsResolvable = assignCallReference?.isError == false
```

Comment:
This is subtle, so I'd like to call out whether it is intentional (and I guess so).

Previously, if `assignCallReference` is null or its `isError` is true, `assignIsError` is true.
Now, only if `assignCallReference` is not null and its `isError` is false, `assignResolvable` is true.

I think the benefit here is, if `assignResolvable` is true, we know `assignCallReference` shouldn't be null? (And that's how `operatorIsResolvable` is used).

Reply:
Yeah you are right. For me I changed this because I find it easier to understand if I assign the boolean flag a positive meaning. Also, note that `assignIsResolvable` is exactly the negate of `assignIsError`.